### DISCUSSION TITLE
Update fix?

### DISCUSSION THREAD

**Discussion by Huntlb** [3kh0/ext-remover#61] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When can we see a fix for the new chrome update? Thanks

**Comment by 3kh0** [comment, marked as answer]
Soon

**Reply by ghost** [reply]
goodness gracious, already at it! good job. Anyways, you spared me putting a keylogger on admin's computer

**Reply by 3kh0** [reply]
@yeeteeyt do it anyway 😈

**Reply by ghost** [reply]
oh nooo don't hack the school admin! that's baaaaaaaad

**Reply by synapt0** [reply]
> Soon

is there an estimated time when its finished?

**Reply by Coolkid560** [reply]
YES LETS GOO! My friend Showed Me This And Now I Need It Back. I Need My School To Get Off My Back, I Know Some Kids Do Bad Things But Now It's Really Annoying. I'm Very Happy That They Are Fixing This.
